flaskr/app.py

```diff
@@ -118,6 +114,19 @@ class Query(graphene.ObjectType):
     node = graphene.relay.Node.Field()
     all_methods = SQLAlchemyConnectionField(MethodObject)
     all_languages = SQLAlchemyConnectionField(LanguageObject)
+    translations = graphene.Field(lambda: graphene.List(MethodResultObject),
+                                  method_id=graphene.Int(required=True),
+                                  target_language_id=graphene.Int(required=True))
+
+    def resolve_translations(parent, info, method_id, target_language_id):
```

Comment:
Interesting ... parent.  This is the first time that I have seen this in python.  Although I am sure it's common.  Getting used to using parent and self.